### PR TITLE
Pokemon Filters Help Modal

### PR DESCRIPTION
**Pull request by versx**
image

image

### CODE REVIEW

**Review by sketchysocks — COMMENTED** [review]
I have mixed feelings about the location of the help button. Maybe have it next to "close" instead? Everything else looks good.

File: src/views/index.mustache [hunk 1]
```diff
@@ -791,3 +845,9 @@
     </div>
 
 {{> footer}}
+
+<script type="text/javascript">
+$(function () {
+    $('[data-toggle="popover"]').popover();
+})
+</script>
```
Comment (Mygod) [inline src/views/index.mustache:853]: It's best to move this to `index.js`.

**Review by Mygod — COMMENTED** [review]
I guess it is better than nothing, although I would make the IV input a dialog with realtime error check/feedback, and put the help text in there using a `<details>`.

File: package.json [hunk 2]
```diff
@@ -27,9 +27,9 @@
     "btoa": "^1.2.1",
     "compression": "^1.7.4",
     "cookie-session": "^1.4.0",
-    "discord-oauth2": "^2.2.0",
-    "discord.js": "^12.2.0",
-    "eslint": "^7.0.0",
+    "discord-oauth2": "^2.5.0",
+    "discord.js": "^12.3.1",
+    "eslint": "^7.9.0",
```
Comment (Mygod) [inline package.json:32]: This change seems unrelated.

File: src/views/index.mustache (hunk 1 above)
Comment (versx) [reply src/views/index.mustache:853]: I need to remove this completely actually, it's not being used, couldn't get the popover to show with being stacked on multiple modals.

**Comment by versx** [comment]
> I have mixed feelings about the location of the help button. Maybe have it next to "close" instead? Everything else looks good.

Moving it next to Close is going to 

> I have mixed feelings about the location of the help button. Maybe have it next to "close" instead? Everything else looks good.

Seems cluttered in my opinion
image

File: src/views/index.mustache (hunk 1 above)
Comment (versx) [inline src/views/index.mustache:853]:
```suggestion
```

**Review by sketchysocks — APPROVED** [review]

**Review by Mygod — APPROVED** [review]
